<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>event_view</class>
 <widget class="QWidget" name="event_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="event_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="user1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>user1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="user2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>user2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="user3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>user3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>390</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>proposedSQL</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="eventListScroll">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>341</width>
     <height>451</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="EventList" name="eventList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>339</width>
      <height>449</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EventList</class>
   <extends>QWidget</extends>
   <header location="global">modules.eventlist.eventlist</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
